core/field_colour.js

```diff
@@ -196,7 +195,7 @@ Blockly.FieldColour.prototype.doClassValidation_ = function(opt_newValue) {
 
 /**
  * Update the value of this colour field, and update the displayed colour.
- * @param {string} newValue The new colour in '#rrggbb' format.
+ * @param {*} newValue The value to be saved. A colour in '#rrggbb' format.
```

Comment:
I think these should say something like "the default validator guarantees that this is a colour in '#rrggbb' format", and similar for the other `doValueUpdate_` functions.

Reply:
Done.